docs/UserGuide.adoc

```diff
@@ -143,11 +149,13 @@ Notifies all members with at least a minimum amount of loyalty points and with t
 
 Format: `notifyEmail [l/LOYALTY_POINTS_MINIMUM] [se/TRUE_FALSE]`
```

Comment:
Are we still supporting this feature????
Didn't we agree to take this feature down?

Reply:
we can put it as [for v2.0]